packages/xsnap/src/avaXS.js

```diff
@@ -112,7 +112,7 @@ async function runTestScript(
   /**
    * Handle callback "command" from xsnap subprocess.
    *
-   * @type { (msg: ArrayBuffer) => Promise<ArrayBuffer> }
+   * @type { (msg: ArrayBuffer) => Promise<Uint8Array<ArrayBufferLike>> }
```

Comment:
This is a significant change of type. New one looks right but I'm surprised we didn't have an error before.